### DISCUSSION TITLE
deprecation message

### DISCUSSION THREAD

**Discussion by lxregistry** [symfony/symfony#50822] [Q&A]
after update to symfony 6.3, this message appear:
The "Monolog\Logger" class is considered final. It may change without further notice as of its next major version. You should not extend it from "Symfony\Bridge\Monolog\Logger".

what I do wrong?

**Comment by ghost** [comment]
Nothing to worry about. Da ya thang and Symfony Core Team will take care of this deprecation here https://github.com/symfony/symfony/issues/47096
